### PR TITLE
Register Screen re commit

### PR DESCRIPTION
**Pull request by Rifaee123**

### CODE REVIEW

**Comment by Rifaee123** [comment]
register added

**Comment by Rifaee123** [comment]
register modified

**Comment by Rifaee123** [comment]
register modifide

File: lib/controller/splash_screen_controller.dart [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import 'package:get/get.dart';
-import 'package:rtd_project/view/login_screen/login_page.dart';
 import 'package:rtd_project/view/register_screen/register_screen.dart';
```
Comment (sreenivas20) [inline lib/controller/splash_screen_controller.dart:3]: splash

**Pull request merged**